src/parser/csql_grammar.y

```diff
@@ -151,8 +151,6 @@ extern int yybuffer_pos;
 
 #define STACK_SIZE	128
 
-#define PT_IS_SERIAL(op)	((op) == PT_NEXT_VALUE || (op) == PT_CURRENT_VALUE)
```

Comment:
This seems to be a fix of a different type of warning (http://jira.cubrid.org/browse/LETS-73)